ote/resources/public/language/sv.edn

```diff
@@ -338,6 +338,7 @@
  :buttons
  {:save "Spara"
   :cancel "Ångra"
+  :continue "Fortsätta"
```

Comment:
Could you add these text to this ticket: https://www.pivotaltracker.com/story/show/163085064

Reply:
Sure, I'm about to create a new ticket about all localisations :)